core/trino-spi/src/test/java/io/trino/spi/TestPage.java

```diff
@@ -160,4 +178,9 @@ private static Block createSlicesBlock(Slice[] values)
         }
         return builder.build();
     }
+
+    private static LazyBlock lazyWrapper(Block block)
+    {
+        return new LazyBlock(block.getPositionCount(), block::getLoadedBlock);
```

Comment:
why not just `() -> block` ?

https://github.com/trinodb/trino/blob/master/core/trino-spi/src/main/java/io/trino/spi/block/LazyBlockLoader.java#L19